### ISSUE TITLE
Failure in rest module when calling filters on EAP-TLS request

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#4657]
**Describe the bug**
When you have the authorize filters enabled (radius_configuration.filter_in_packetfence_authorize) and you perform an EAP-TLS request, you will obtain a failure related to the length of the encoding of the EAP-Message resulting in an invalid JSON payload being sent to httpd.aaa which results in a 415 and FreeRADIUS rejecting the user

**To Reproduce**
1. Enable radius_configuration.filter_in_packetfence_authorize
2. Restart radiusd-auth
3. Perform an EAP-TLS request (@lzammit has a cert/key/CA that is problematic if needed)
4. Logs should show the following failure:
```
Jul  9 11:39:06 zammitcorp-otbm-eap-tls auth[20918]: (5) rest: ERROR: {"error":{"detail":"malformed JSON string, neither tag, array, object, number, string or atom, at character offset 950 (before \"(end of string)\") at /usr/local/pf/lib/pf/WebAPI/R
EST.pm line 43.\n","message":"Cannot parse request"}}
Jul  9 11:39:06 zammitcorp-otbm-eap-tls auth[20918]: (5) Invalid user (rest: AVP exceeds buffer length or chunk): [mpf@otbm.com] (from client 10.3.0.6 port 13 cli 00:11:22:33:44:55)
Jul  9 11:39:06 zammitcorp-otbm-eap-tls auth[20918]: [mac:00:11:22:33:44:55] Rejected user: mpf@myvest.com
```

**Expected behavior**
Should perform the filtering and not cause the request to fail

**Comment by fdurand** [comment]
It looks to be the EAP-Message that is too big to be encoded.

**Comment by julsemaan** [comment]
Its like trying to fit @lzammit in this car:
![image](https://user-images.githubusercontent.com/3857942/61225831-5da50080-a6ef-11e9-8e1f-753c74199939.png)

**Comment by fdurand** [comment]
Fixes by a8d9110046a60560ca935f628651ebc64ff76e44
